client/src/js/components/bhRequiredInventoryScanSelect/bhRequiredInventoryScanSelect.js

```diff
@@ -0,0 +1,39 @@
+angular.module('bhima.components')
+  .component('bhRequiredInventoryScanSelect', {
+    templateUrl : 'js/components/bhRequiredInventoryScanSelect/bhRequiredInventoryScanSelect.tmpl.html',
```

Comment:
By the way, if you ever want to unit test this, the `templateUrl` should just be `bhRequiredInventoryScanSelect/bhRequiredInventoryScanSelect.tmpl.html`.  For some reason, Karma doesn't find the full path referencing `js/components`.

Reply:
I tried this but it broke the interface. I reverted it to the way it was.
Perhaps Karma is looking in the old default directory:  modules/templates?